src/controllers/reportController.js

```diff
@@ -0,0 +1,167 @@
+import { User, Report, Article } from '../sequelize/models';
+
+/**
+ * @description user should be able to report an article
+ */
+class reportArticleController {
+  /**
+   *
+   * @param {object} req
+   * @param {object} res
+   * @returns {object} return object containg data of saved report
+   */
+  static async createReport(req, res) {
+    const userId = req.userData.id;
+    const { articleSlug } = req.params;
+    const { reason, description } = req.body;
+
+    const authorProfile = await User.findOne({ where: { id: userId } });
+    const findArticle = await Article.findOne({ where: { slug: articleSlug } });
```

Comment:
`article` is a better name. Verbs e.g `find` are more suited for functions/methods

Reply:
done